ergotree-interpreter/src/sigma_protocol/prover.rs

```diff
@@ -843,7 +843,7 @@ fn step9_real_threshold(ct: CthresholdUnproven) -> Result<Option<ProofTree>, Pro
                 ProofTree::UnprovenTree(unpt) => unpt.challenge(),
             };
             if let Some(challenge) = challenge_opt {
-                points.append(&mut one_based_idx.to_be_bytes().to_vec());
```

Comment:
Revert this to the original line.

Reply:
Original line causes the following error:

`Gf2_192PolyError(InterpolatePointsAndValuesLengthDiffer)`

Reply:
Your change here is correct. Each point has to be a distinct `u8`.